components/Starknet/modules/architecture-and-concepts/pages/network-architecture/transactions.adoc

```diff
@@ -462,6 +462,11 @@ The generator used in the ECDSA algorithm is stem:[G=\left(g_x, g_y\right)] wher
 
 stem:[g_x=874739451078007766457464989774322083649278607533249481151382481072868806602] stem:[g_y=152666792071518830868575557812948353041420400780739481342941381225525861407]
 
+[id="v3-fee-estimation"]
+== v3 transaction fee estimation
+
+Estimate the fees of transactions with the `starknet_estimateFee` API call, which is part of Starknet's JSON RPC v0.7.0 and above. For more information, see the link:https://github.com/starkware-libs/starknet-specs/blob/v0.7.1/api/starknet_api_openrpc.json#L612[Starknet JSON RPC specification]. For more information on how to construct the appropriate `resource_bounds` based on the response of `starknet_estimateFee`, see link:https://community.starknet.io/t/starknet-v0-13-1-pre-release-notes/113664#sdkswallets-how-to-use-the-new-fee-estimates-7[How to use the new fee estimates?] on the Starknet community forum.
```

Comment:
JSON RPC is an API format. Referring to Starknet's API as "Starknet's JSON RPC" is grammatically incorrect.